NineChronicles.Headless.Executable/Program.cs

```diff
@@ -169,9 +166,10 @@ static void ConfigureSentryOptions(SentryOptions o)
 #endif
             
             // Setup logger.
+            var builder = new ConfigurationBuilder().AddJsonFile("appsettings.json");
+            var configuration = builder.Build();
```

Comment:
How about renaming into `configurationBuilder`?

```suggestion
            var configurationBuilder = new ConfigurationBuilder().AddJsonFile("appsettings.json");
            var configuration = configurationBuilder.Build();
```